kv/dist_sender.go

```diff
@@ -41,9 +41,7 @@ import (
 
 // Default constants for timeouts.
 const (
-	// TODO(bdarnell): make SendNextTimeout configurable.
-	// https://github.com/cockroachdb/cockroach/issues/6719
-	defaultSendNextTimeout = 500 * time.Millisecond
+	defaultSendNextTimeout = 10 * time.Second // for now; see #2500
```

Comment:
Can you update the issue number? There should be a TODO here that explains this is possibly masking a bug.